cmd/modelcmd/controller.go

```diff
@@ -19,19 +19,18 @@ var (
 	// ErrNoControllersDefined is returned by commands that operate on
 	// a controller if there is no current controller, no controller has been
 	// explicitly specified, and there is no default controller.
-	ErrNoControllersDefined = errors.New(`no controller
+	ErrNoControllersDefined = errors.New(`No controllers registered.
 
-Please either create your own new controller using "juju bootstrap" or
-connect to another controller that you have been given access to using "juju register".
+Please either create a new controller using "juju bootstrap" or connect to
+another controller that you have been given access to using "juju register".
 `)
```

Comment:
I'm pretty sure we shouldn't try and fix this here, but we should define error messages as simple strings with no newlines, and the view on the error should be controlled when its output.